doc/cygnus-ngsi/installation_and_administration_guide/backends_as_sth.md

```diff
@@ -0,0 +1,40 @@
+#<a name="top"></a>Backends as short-term historics
+Backends are used by Cygnus NGSI as "infinite" context data repositories. More and more data is appended to files, tables and collections as data flows from a NGSI source. Such a data flow may never end, thus, insertions may never end too, exhausting the available storing resources.
+
+Therefore, it is important to provide mechanisms in charge of controlling how much data is stored in the persistence backends, removing old data in favour of new one, resulting in some kind of short-term historic implementation. 
+
+##<a name="top"></a>How it works
+There are two approaches when deciding which data must be removed from existent historics:
+
+* By <b>capping</b> data "records"(*) once certain size limit has been reached. In other words, to control only the last N records are stored.
+* By <b>expirating</b> "records"(*) once certain keepalive limit has been reached. In other wods, to control only records added in the last N seconds are stored.
+
+The above can be controlled by means of specific configuration parameters in certain sinks:
+
+| Parameter | Mandatory | Default value | Comments |
+|---|---|---|---|
+| persistence\_policy.max_records | no | -1 | Maximum number of records allowed for a table before it is capped. `-1` disables this policy. |
+| persistence\_policy.expiration_time | no | -1 | Maximum number of seconds a record is maintained in a table before expiration. `-1` disables this policy. |
+| persistence\_policy.checking_time | no | 3600 | Frequency (in seconds) at which the sink checks for record expiration. |
+
+Which sinks provide this kind of functionality? For the time being:
+
+* `NGSIMySQLSink`
+* `NGSICKANSink`
+
+(*) A record may mean many different things depending on the persistence backend: a Json entry in a HDFS file, a row in a MySQL table, a record in a CKAN resoeurce...
+
+[Top](#top)
+
+##<a name="top"></a>The special case of `NGSIMongoSink` and `NGSISTHSink`
+`NGSIMongoSink` and `NGSISTHSink` already implemented this kind of functionality, since the data stored in MongoDB and STH Comet was wanted to be a short-term historic from the very begining. In these cases, the parameters controlling the functionality are:
```

Comment:
There is no conflict at all :) The capping and/or expirating records feature is only for CKAN and MySQL (as already stated at the end of the previpus section, immediatelly after the table). By adding this section I wanted to remember something similiar already existed for MongDB/STH, but I wanted to highlight the differeces as well. I'll pecifically comment about this lack of potential conflict :)

Anyway, something I would like to do is, in the future, to homogenize both ways (CKAN and MySQL on one side, MongoDB/STH on the other side) of configuring ta feature that is conceptually the same. I.e. I would like to use the same configuration parameters.